### PR TITLE
Nttw5 updates

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig [hunk 1]
```diff
@@ -6,12 +6,15 @@
...
 
 <h2>When</h2>
-<p>We are currently exploring dates in late 2021.</p>
-<p><i>Note: No Time To Wait had originally been scheduled for September 2020 and later postponed to March 2021; however, due to the ongoing coronavirus (COVID-19) pandemic, No Time to Wait 5 is yet unscheduled. We continue to follow the developments of the pandemic and hope to provide a schedule when it's safe to do so.</i><p>
+<p>December 8-10, 2021</p>
+<p><i>Note: Due to the ongoing coronavirus (COVID-19) pandemic, No Time To Wait, which was originally scheduled for September 2020, had a series a postponements. As scientific and social progress continues to address the pandemic, we continue to follow policy developments and are hopeful that we can hold a safe and successful conference.</i><p>
```
Comment (retokromer) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig]: nit: missing slash at closing tag `</p>`

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig [hunk 2]
```diff
@@ -44,12 +47,6 @@
 <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2443.436270359645!2d5.170862851972044!3d52.23545897966196!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47c66b5af17f1a73%3A0xd62b509149ca15ac!2sNetherlands%20Institute%20for%20Sound%20and%20Vision!5e0!3m2!1sen!2sus!4v1579973749555!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
 </p>
 
-<p>Parts of the conference will also be accessible remotely.</p>
-
-<h2>Timeline</h2>
-
-<p>We continue to monitor the development and impact of the coronavirus and actively seek out health and safety guidance. If we can proceed with our plans safely, we hope that we can open the call for proposals and registration in September 2020.</p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:51]: "September 2020", oops, good to remove this outdated expected date...

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig [hunk 3]
```diff
@@ -6,12 +6,15 @@
...
 
 <h2>When</h2>
-<p>We are currently exploring dates in late 2021.</p>
-<p><i>Note: No Time To Wait had originally been scheduled for September 2020 and later postponed to March 2021; however, due to the ongoing coronavirus (COVID-19) pandemic, No Time to Wait 5 is yet unscheduled. We continue to follow the developments of the pandemic and hope to provide a schedule when it's safe to do so.</i><p>
+<p>December 8-10, 2021</p>
+<p><i>Note: Due to the ongoing coronavirus (COVID-19) pandemic, No Time To Wait, which was originally scheduled for September 2020, had a series a postponements. As scientific and social progress continues to address the pandemic, we continue to follow policy developments and are hopeful that we can hold a safe and successful conference.</i></p>
+<p>Call for proposals and conference registration coming soon.</p>
+
+<h2>Survey</h2>
+<p>We are currently developing a timeline to announce a call for proposals and open registration soon. To help us prepare for this, we request that you complete <a href="http://bit.ly/nttw5prelimsurvey">our preliminary survey</a>.</p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:17]: The "What" title (see nttw4 page) disappeared in a previous patch, would be good to have it back.

image

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig [hunk 4]
```diff
@@ -6,12 +6,17 @@
...
 
 <h2>When</h2>
-<p>We are currently exploring dates in late 2021.</p>
-<p><i>Note: No Time To Wait had originally been scheduled for September 2020 and later postponed to March 2021; however, due to the ongoing coronavirus (COVID-19) pandemic, No Time to Wait 5 is yet unscheduled. We continue to follow the developments of the pandemic and hope to provide a schedule when it's safe to do so.</i><p>
+<p>December 8-10, 2021</p>
+<p><i>Note: Due to the ongoing coronavirus (COVID-19) pandemic, No Time To Wait, which was originally scheduled for September 2020, had a series a postponements. As scientific and social progress continues to address the pandemic, we continue to follow policy developments and are hopeful that we can hold a safe and successful conference.</i></p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig]: "a series a postponements" ?

Comment (dericed) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig]: fixed of `of`

**Review by JeromeMartinez — APPROVED** [review]

**Review by retokromer — APPROVED** [review]
Fingers crossed.

**Pull request merged**